e2e/wdio/core/tests/time.e2e-spec.ts

```diff
@@ -0,0 +1,107 @@
+import { TimePo } from '../pages/time.po';
+import {
+    click,
+    clickNextElement,
+    clickPreviousElement,
+    getNextElementText,
+    getPreviousElementText,
+    getText,
+    pause,
+    refreshPage,
+} from '../../driver/wdio';
+import { sections } from '../fixtures/appData/time-contents'
+
+describe('Time component test', function () {
+    const timePage = new TimePo();
+    const {
+        downArrow,
+        UpArrow,
+        hoursColumn,
+        minutesColumn,
+        secondsColumn,
+        currentHour,
+        currentMinute,
+        currentSec,
+        noSpinnersExample,
+        programmaticallyExample,
+        withoutSecondsExample,
+        onlyHoursExample,
+        set11HoursBtn,
+        formExample,
+        timeItem,
+        enableTimeRow
+    } = timePage;
+
+    beforeAll(() => {
+        timePage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+    }, 2);
+
```

Comment:
think you could add these cases:
click and drag (you can click and drag the hours/minutes/seconds)
check 12 hour clock is 12 hour
change am/pm on 12 hour clock
check 24 hour clock is 24 hour
check no spinner is without up/down arrows
sizes example has dynamic content- can't use visual regression. need to check compact is compact, cozy is cozy.